src/generated/resources/data/forge/tags/biomes/wet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:swamp",
+    "minecraft:jungle",
+    "minecraft:jungle_hills",
+    "minecraft:jungle_edge",
+    "minecraft:swamp_hills",
+    "minecraft:modified_jungle"
```

Comment:
Any reason why `modified_jungle_edge` is not here?
Also might want to include `bamboo_jungle` and `bamboo_jungle_hills`